Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/Cql2ElmVisitor.java

```diff
@@ -3368,8 +3368,10 @@ else if (classType.getPrimaryCodePath() != null) {
             // Resolve the terminology target using an in or ~ operator
             try {
                 if (codeComparator == null) {
-                    codeComparator = terminology.getResultType() instanceof ListType ||
-                            terminology.getResultType().isSubTypeOf(libraryBuilder.resolveTypeName("System", "Vocabulary")) ? "in" : "~";
+                    codeComparator = ((!libraryBuilder.isCompatibleWith("1.5") && terminology.getResultType() instanceof ListType)
+                            || (libraryBuilder.isCompatibleWith("1.5")
+                                    && terminology.getResultType().isSubTypeOf(libraryBuilder.resolveTypeName("System", "Vocabulary"))))
+                            ? "in" : "~";
```

Comment:
I see the case that it's 1.5 and a Vocabulary type, and I see the case where it's 1.4 and a List type. 1.4 and Vocabulary type isn't possible (because Vocabulary is 1.5). Is 1.5 and List type possible? I'm thinking cases where the ValueSet is expanded.

Reply:
Yes, good point, relaxed.